packages/bootstrap-vue-next/src/types/BreakpointProps.ts

```diff
@@ -19,11 +19,11 @@ export interface OrderBreakpointProps {
 }
 
 export interface ColBreakpointProps {
-  sm?: LiteralUnion<Booleanish, Numberish>
-  md?: LiteralUnion<Booleanish, Numberish>
-  lg?: LiteralUnion<Booleanish, Numberish>
-  xl?: LiteralUnion<Booleanish, Numberish>
-  xxl?: LiteralUnion<Booleanish, Numberish>
+  sm?: LiteralUnion<boolean, Numberish>
+  md?: LiteralUnion<boolean, Numberish>
+  lg?: LiteralUnion<boolean, Numberish>
+  xl?: LiteralUnion<boolean, Numberish>
+  xxl?: LiteralUnion<boolean, Numberish>
```

Comment:
```suggestion
  sm?: boolean | Numberish
  md?: boolean | Numberish
  lg?: boolean | Numberish
  xl?: boolean | Numberish
  xxl?: boolean | Numberish
```

Literal union here has no effect